Directory.Build.props

```diff
@@ -44,4 +44,7 @@
       <AdditionalIncludeDirectories>$(SolutionDir)\build\override;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
     </Midl>
   </ItemDefinitionGroup>
+
+  <!-- Build with the hybrid CRT (Universal CRT + Static VS CRT (for what little the Universal CRT doesn't cover) -->
+  <Import Project="$(MSBuildThisFileDirectory)HybridCRT.props" />
```

Comment:
>HybridCRT [](http://example.com/codeflow?start=46&length=9)

there is some insistency with EnsureOutputLayout.props. That one is under build folder.

Reply:
That's the only one under build, all other *.props are in the root. And it's unclear if "build" is for all builds or pipeline builds, so better to stick with the crowd and have someone and have someone (sooner or later) take a holistic look